docs/GettingStarted/03-What-is-a-Fedimint.md

```diff
@@ -1,79 +1,79 @@
+It makes sense to consider Fedimints deployed on a small scale for example, as a community bank for a small community with a robust circular economy and remember there will be many Fedimints deployed on the lightning network and not a single global implementation.
 
-The open protocol enables communities of Bitcoiners to come together at their own discretion, assist each other with their custody and backup arrangements and make lightning payments through a privacy preserving wallet. 
+The open protocol enables communities of Bitcoiners to come together at their own discretion, assist each other with their custody and backup arrangements and make lightning payments through a privacy preserving wallet.
 
-Run for the community, by the community.
+Fedimints are run for the community, by the community.
 
 ## Fedimint Components Structure
+
 A Fedimint has five functional components (accounts, custody & redemption, backup & recovery, transaction processing, LN Gateway) and three user personas (Guardian, User, Lightning Gateway) as outlined in the figure and with details below.
 
 ![High level logical model of a fedimint](/img/Fedimint-HL.png)
 
+All of these are supported by three core technology components, discussed in greater detail in ["Overview > Technologies used in Fedimint"](TechCompontents).
 
-All of these are supported by three core technology components eCash, lighting discussed in more detail in ["Overview > The Technology Components"](TechCompontents).
+### Fedimint Guardians
 
-### Federation Guardians
-Federation guardians are the technical community members responsible for setting up and running the Fedimint.  
+Fedimint guardians are the technical community members responsible for setting up and running the Federated Chaumian eCash system. They custody and backup funds and are responsible for the minting and redemption of eCash token claims against the Bitcoin reserves. A fedimint member is explicitly trusting that the federation guardians will not form a majority quorum to collude to steal user funds.
 
-The guardians will coordinate using a [consensus protocol (HBBFT)](../CommonTerms/HBBFTConsensus) which is part of the Fedimint software deployed on personal servers.  This means that their computers will need to be on the same network and will be "talking" constantly in order to process the various transactions of the Fedimint.
+The guardians coordinate using a [distributed consensus protocol (HBBFT)](../CommonTerms/HBBFTConsensus) which is part of the Fedimint software deployed on personal servers. The consensus mechanism is Asynchronous, Byzantine Fault Tolerant, and can survive and operate even if a minority of individual federation guardians disconnect from the network. If a majority of the federation guardians disconnect, the consensus protocol will halt execution until a quorum of guardians come back online, at which point the consensus will continue.
```

Comment:
Don't disagree but may go through this after merging to simplify the langauge and we can offload some of the more technical details into the FAQ on what is HBBFT.